src/osara.h

```diff
@@ -159,6 +161,7 @@ typedef struct Command {
 	void (*execute)(Command*);
 } Command;
 extern int lastCommandRepeatCount;
+extern Command* lastCommand ;
```

Comment:
I don't think this is needed in this patch? You're accessing lastCommand from within reaper_osara.cpp where it is defined.